packages/polyfills/src/url.node.ts

```diff
@@ -1,13 +1 @@
-import {URL, URLSearchParams} from 'url';
-
-declare global {
-  namespace NodeJS {
-    export interface Global {
-      URL: typeof URL;
-      URLSearchParams: typeof URLSearchParams;
-    }
-  }
-}
-
-global.URL = URL;
-global.URLSearchParams = URLSearchParams;
+// URL and URLSearchParams available as globals by default.
```

Comment:
Add `"node": ">=10.0.0"` to this package's `package.json#engines`.

Reply:
Discussed on Slack: going to leave this off for now because both Travis and Shipit are on Node 8.x :(